emottak-utils/src/main/kotlin/no/nav/emottak/utils/kafka/KafkaPublisherClient.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.emottak.utils.kafka
+
+import no.nav.emottak.utils.config.Kafka
+import no.nav.emottak.utils.config.toProperties
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.common.serialization.StringSerializer
+
+class KafkaPublisherClient(val topic: String, val config: Kafka) : AutoCloseable {
+    // private val log = LoggerFactory.getLogger(this.javaClass)
```

Comment:
Du kan fjerne alt ubrukt kode i denne filen

Reply:
Det er veldig bra innsats, men det ser ut som denne PR-en kanskje ikke er helt ferdig ennå? I slike tilfeller er det bedre å lage en draft PR, slik at vi kan diskutere løsningen uten at det skaper unødvendig frem-og-tilbake i code review.

Generelt er det et par tommelfingerregler som kan være nyttige her:

- Unngå å legge igjen TODO-kommentarer i koden. Det finnes mye informasjon om hvorfor dette anses som en code smell, men kort sagt er det bedre å følge opp slike ting i oppgaver eller dokumentasjon.
- Unngå ubrukt kode; det gjør koden ryddigere og lettere å vedlikeholde.
- Bruk draft PR hvis koden ikke er klar for review, men du likevel ønsker tilbakemelding eller diskusjon.

Hvordan ønsker du å løse loggingen her?